pkg/features/features.go

```diff
@@ -56,13 +56,6 @@ func SnowProvider() Feature {
 	}
 }
 
-func NutanixProvider() Feature {
-	return Feature{
-		Name:     "Nutanix provider support",
-		IsActive: globalFeatures.isActiveForEnvVar(NutanixProviderEnvVar),
-	}
-}
```

Comment:
Why is this being removed?

Reply:
this was duplicate. pls see line 74 below